ArjunaJTA/jta/classes/com/arjuna/ats/jta/logging/jtaI18NLogger.java

```diff
@@ -580,10 +580,14 @@
     @Message(id = 16142, value = "Not actived transaction ''{0}'' for xid: {1}", format = MESSAGE_FORMAT)
     String get_not_activated_transaction(Transaction txn, Xid xid);
 
-    @Message(id = 16143, value = "Problem during waiting for lock '{0}' whilst in state {1}", format = MESSAGE_FORMAT)
+    @Message(id = 16143, value = "Problem during waiting for lock ''{0}'' whilst in state {1}", format = MESSAGE_FORMAT)
```

Comment:
Won't this change affect any existing users that are potentially parsing this output? If so you should deprecate first.

Reply:
This fixes a formatting bug.
The use of `'{0}'` causes that data are not pasted there. Aka the call to method `public void warn_intteruptedExceptionOnWaitingXARecoveryModuleLock(XARecoveryModule recModule, String state, ...` expects to inform user about what recovery module is in use. Now the recovery module is **not** printed. The use of `'{0}'` causes the data is not placed to the output. The change to `''{0}''` makes the information to be printed.

Reply:
OK